### PR TITLE
feat: get invoices

### PR DESCRIPTION
**Pull request by enguerran**
With this PR, the connector is able to login to the website and fetch the invoices.
Once invoices are fetched, they are saved as bills in cozy.

### CODE REVIEW

**Comment by enguerran** [comment]
You could checkout this PR and try it.

__known issue:__ 
- [ ] the pdf is badly created cf. https://github.com/konnectors/libs/pull/259 (need to update the lib)

**Comment by doubleface** [comment]
cozy-konnector-libs 4.5.1 is released https://github.com/konnectors/libs/releases/tag/cozy-konnector-libs%404.5.1

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```
Comment (doubleface) [inline .vscode/launch.json]: I think this should not be saved in the repository

File: src/index.js [hunk 2]
```diff
@@ -4,124 +4,125 @@ const {
...
-        parse: src => `${baseUrl}/${src}`
+        sel: 'td:nth-child(4)',
+        fn: extractText,
+        parse: text => /(\d+,\d+)/g.exec(text)[0]
       },
-      filename: {
-        sel: 'h3 a',
-        attr: 'title',
-        parse: title => `${title}.jpg`
+      numero: {
```
Comment (doubleface) [inline src/index.js]: The amount must be a number. Please add a parseFloat.

File: src/index.js [hunk 3]
```diff
@@ -4,124 +4,125 @@ const {
...
   const documents = await parseDocuments($)
 
-  // here we use the saveBills function even if what we fetch are not bills, but this is the most
-  // common case in connectors
   log('info', 'Saving data to Cozy')
   await saveBills(documents, fields.folderPath, {
-    // this is a bank identifier which will be used to link bills to bank operations. These
-    // identifiers should be at least a word found in the title of a bank operation related to this
-    // bill. It is not case sensitive.
     identifiers: ['books']
```
Comment (doubleface) [inline src/index.js]: This should be a bank identifier, a word found in the associated bank operation which is specific to this vendor.

File: src/index.js [hunk 4]
```diff
@@ -4,124 +4,125 @@ const {
...
 
-  // here we use the saveBills function even if what we fetch are not bills, but this is the most
-  // common case in connectors
   log('info', 'Saving data to Cozy')
   await saveBills(documents, fields.folderPath, {
-    // this is a bank identifier which will be used to link bills to bank operations. These
-    // identifiers should be at least a word found in the title of a bank operation related to this
-    // bill. It is not case sensitive.
-    identifiers: ['books']
+    identifiers: ['monde-diplomatique']
```
Comment (aenario) [inline src/index.js]: There is a hyphen `-` in the bank operation name ?

Comment (enguerran) [reply src/index.js]: I have no idea

**Pull request merged**